MaxText/input_pipeline/_tfds_data_processing.py

```diff
@@ -187,7 +191,7 @@ def get_datasets(
   # eval_data = get_raw_dataset(eval_ds_builder, config.eval_split)
   eval_ds = eval_ds_builder.as_dataset(split=config.eval_split,
                                           read_config = read_config,
-                                          shuffle_files=config.enable_data_shuffling)
+                                          shuffle_files=False)
```

Comment:
Wait we want to offer this still right but set to False? am I missing something?

Reply:
Thank you Rafi, if I am understanding correctly, evaluation dataset is usually not recommended to be shuffled since it won't help in back prop and it might introduce subtle differences from different orderings between 2 eval runs. Not an issue for c4_mlperf eval dataset which loads processed data directly.

I found it when checking if we can get the same first eval batch between 2 different eval runs.

Reply:
my mistake, thank you